artman/utils/protoc_utils.py

```diff
@@ -298,11 +298,25 @@ def prepare_pkg_dir(output_dir, api_name, api_version, organization_name,
     return pkg_dir
 
 
-def find_proto_files(proto_path):
-    for dirpath, subdirs, files in os.walk(proto_path):
-        for f in files:
-            if f.endswith('.proto'):
-                yield os.path.join(dirpath, f)
+def find_protos(proto_paths, excluded_proto_path):
+    """Searches along `proto_path` for .proto files and returns a generator of
+    paths"""
+    if type(proto_paths) is not list:
```

Comment:
Python complains that collections module does not have `Generator` so I use `types.GeneratorType` instead. Consider done.